include/quda.h

```diff
@@ -1717,7 +1717,7 @@ extern "C" {
    * @param[in,out] obs_param Parameter struct that defines which
    * observables we are making and the resulting observables.
    */
-  void performAdjGFlowSafe(void *h_out, void *h_in, QudaInvertParam *inv_param, QudaGaugeSmearParam *smear_param);
+  void performAdjGFlowSafe(void **h_out, void **h_in, QudaInvertParam *inv_param, QudaGaugeSmearParam *smear_param, const size_t nSpinors);
```

Comment:
The doxygen doesn't match the function signature, and the `nSpinors` argument needs to be added to the doxygen.

Reply:
Also, `nSpinors` does not need to be `const` here since this is passed by value.